<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWidget</class>
 <widget class="QWidget" name="CalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>844</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_3" rowminimumheight="0">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item row="0" column="1">
      <widget class="AdvancedImageWidget" name="imageWidget" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>298</width>
          <height>759</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QGroupBox" name="imageBox">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Images</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>8</number>
            </property>
            <property name="topMargin">
             <number>8</number>
            </property>
            <property name="rightMargin">
             <number>8</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="inputSettingsButton">
        <property name="text">
         <string>Input</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/webcam.png</normaloff>:/new/prefix1/webcam.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="camerasSettingsButton">
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/2cam-blend-b.png</normaloff>:/new/prefix1/2cam-blend-b.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startRecordingButton">
        <property name="toolTip">
         <string>Start Recording</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/play.png</normaloff>:/new/prefix1/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopRecordingButton">
        <property name="toolTip">
         <string>Stop Recording</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/stop_2.png</normaloff>:/new/prefix1/stop_2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseRecordingButton">
        <property name="toolTip">
         <string>Pause Recording</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/pause.png</normaloff>:/new/prefix1/pause.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="value">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start Calibration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopShowing">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop Showing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveCalib">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Save Calibration</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">advancedImageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>addImage()</slot>
  <slot>startCalibration()</slot>
  <slot>startRecording()</slot>
  <slot>updateVideo()</slot>
  <slot>stopShowing()</slot>
  <slot>saveMatrix()</slot>
 </slots>
</ui>
